var/spack/repos/builtin/packages/netcdf-fortran/package.py

```diff
@@ -116,6 +122,41 @@ def configure_args(self):
             config_args.append('FC=%s' % self.spec['mpi'].mpifc)
             config_args.append('F77=%s' % self.spec['mpi'].mpif77)
 
+        if '+dap' in self.spec:
+            config_args.append('--enable-dap')
+        else:
+            config_args.append('--disable-dap')
+
+        if '+pic' in self.spec:
+            config_args.append('--with-pic')
+        else:
+            config_args.append('--without-pic')
+
+        if '+jna' in self.spec:
+            config_args.append('--enable-jna')
+        else:
+            config_args.append('--disable-jna')
+
+        if '+pnetcdf' in self.spec:
+            config_args.append('--enable-pnetcdf')
+        else:
+            config_args.append('--disable-pnetcdf')
+
+        if '+netcdf4' in self.spec:
+            config_args.append('--enable-netcdf-4')
+        else:
+            config_args.append('--disable-netcdf-4')
+
+        if '+ncgen4' in self.spec:
+            config_args.append('--enable-ncgen4')
+        else:
+            config_args.append('--disable-ncgen4')
+
+        if '+doxygen' in self.spec:
+            config_args.append('--enable-doxygen')
+        else:
+            config_args.append('--disable-doxygen')
```

Comment:
We can keep this if `doxygen` is listed as a `build` dependency of the package (not the case now). By the way, if the author decides to keep the name of the variant, this should become
```python
config_args += self.enable_or_disable('doxygen')
```